tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,1023 @@
+              <param name="t_min" type="float" optional="true" label="Start time in MJD" />
+              <param name="t_max" type="float" optional="true" label="Stop time in MJD" />
+              <!-- Dataproduct_type field values definition https://www.ivoa.net/rdf/product-type/2023-06-26/product-type.html  -->
+              <conditional name="dataproduct_type_selection">
+                <param name="dataproduct_input_type" type="select" label="Data product type input choice">
+                  <option value="list" selected="true">From list</option>
+                  <option value="custom">Custom (Input data product type yourself)</option>
+                </param>
+                <when value="list">
+                  <param name="dataproduct_type_select" type="select" label="Data product type" help="Logical data product type (link to reference in Help section)">
+                    <option value="none" selected="true">None</option>
+                    <option value="custom">Custom (Input data product type yourself)</option>
+                    <option value="image">Image</option>
+                    <option value="cube">Cube</option>
+                    <option value="dynamic-spectrum">Dynamic spectrum</option>
+                    <option value="event-list">Event list</option>
+                    <option value="light-curve">Light curve</option>
+                    <option value="measurements">Measurements</option>
+                    <option value="polarization-cube">Polarization cube</option>
+                    <option value="polarization-resolved-dataset">Polarization resolved dataset</option>
+                    <option value="polarized-spectrum">Polarized spectrum</option>
+                    <option value="sed">SED</option>
+                    <option value="slit-spectrum">Slit spectrum</option>
+                    <option value="spatially-resolved-dataset">Spatially resolved dataset</option>
+                    <option value="spectral-cube">Spectral cube</option>
+                    <option value="spectrally-resolved-dataset">Spectrally resolved dataset</option>
+                    <option value="spectrum">Spectrum</option>
+                    <option value="temporally-resolved-dataset">Temporally resolved dataset</option>
+                    <option value="time-cube">Time cube</option>
+                    <option value="timeseries">Time series</option>
+                    <option value="velocity-curve">Velocity curve</option>
+                    <option value="visibility">Visibility</option>
+                  </param>
+                </when>
+                <when value="custom">
+                  <param name="dataproduct_type" type="text" label="Data product type" help="Logical data product type" />
+                </when>
+              </conditional>
+              <param name="obs_collection" type="text" label="Collection" help="Name of the observation collection" />
```

Comment:
I guess the description could be the label and you can drop the help

Reply:
```suggestion
              <param name="obs_collection" type="text" label="Observation collection name"/>
```